app/src/main/java/org/mozilla/fenix/compose/ImagesPlaceholder.kt

```diff
@@ -0,0 +1,87 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.painter.ColorPainter
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import mozilla.components.support.images.compose.loader.Fallback
+import mozilla.components.support.images.compose.loader.ImageLoaderScope
+import mozilla.components.support.images.compose.loader.Placeholder
+import mozilla.components.ui.colors.PhotonColors
+
+/**
+ * Renders the app default image placeholder while the image is still getting loaded.
+ *
+ * @param modifier [Modifier] allowing to control among others the dimensions and shape of the image.
+ * @param contentDescription text provided to accessibility services to describe what this image represents.
```

Comment:
Just a general feedback that I noticed with all the @param. I would generally just start with capitalizing so it doesn't end up looking a bit awkward when we have multiple sentences for the @param description.


```suggestion
 * @param contentDescription Text provided to accessibility services to describe what this image represents.
```

Reply:
Done.